profile_collection/startup/00-startup.py

```diff
@@ -29,6 +29,77 @@
 pd.options.display.max_columns = 10
 
 
+from metadatastore.mds import MDS
+from databroker import Broker
+from databroker.core import register_builtin_handlers
+from filestore.fs import FileStore
+
+_mds_config = {'host': 'xf03id-ca1',
```

Comment:
This is a step forward, and could be merged as is, but we are moving toward using yml files like `/etc/databroker/hxn.yml`. We laid out the provision plan for doing this [here](https://github.com/NSLS-II/databroker/issues/194) and documented the final version [here](https://nsls-ii.github.io/databroker/configuration.html).